roles-in-tech/dev-ops.md

```diff
@@ -17,75 +13,84 @@ Here are links to lessons that should be completed before this lesson.
+- [Selenium](https://www.seleniumhq.org/) is an automated browser that allows QA teams to write scripts and test web products. It’s compatible with eight popular programming languages.
 
-Build on the first information. Have apprentices guess things, do an activity, etc.
+- *For better understanding of **DevOps** refer to this slide [Roles in Tech:DevOps](https://docs.google.com/presentation/d/1GaoHCdvQ2UplDjapp2wG-VRsxlqtzFqIpOtIqvKNYOo/edit?usp=sharing)*
 
-Make sure to mention these things:
+### Lesson
 
-- Things
-	- This is a sub-thing
-- More things
-- Even more things
-- Even more things
+**_DevOps_** stands for development and operations. It’s a practice that aims at merging development, quality assurance, and operations (deployment and integration) into a single, continuous set of processes. This methodology is a natural extension for Agile and continuous delivery approaches.
 
 
 ### Common Mistakes / Misconceptions
 
-List things that apprentices might not realize, might assume at first, or should avoid.
+While many industry leaders have successfully deployed DevOps to achieve faster speed-to-market with higher quality software and a first-class customer experience, there continues to be some fundamental misconceptions about what DevOps really means.
 
-- Example
-- Example
+- **DevOps is a set of automated tools.**
+- **DevOps procedures change from project to project.**: 
 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
+DevOps consists of a lot of tools. From learning *Git VCS* to deploying code in *Docker*, one need to learn a variety of skills to become a good DevOps Engineer.
+[Edureka](https://www.edureka.co/devops)
 
 ### Independent Practice
 
-Class does this thing themselves with specific additional items. This could be alone, with a partner, or small group; but the idea is that it's less guided, more independent.
+Spend 5 minutes on [DevOps best practices](https://devops.com/design-devops-best-practices/)
 
 ### Challenge
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
-
+To become a certified DevOps Engineer one should refer to [AWS Certified DevOps Engineer - Professional](https://aws.amazon.com/certification/certified-devops-engineer-professional/)
 
 ### Check for Understanding
 
-Some ideas: have apprentices summarize to each other, make a cheat sheet, take a quiz, do an assignment, or something else that helps them self-assess their understanding. This exercise should help apprentices determine whether they've met the outline objectives, or if they need to review.
+For Better Understanding of *DevOps* , one should do continuous unit testing after deploying it on a suitable platform.
```

Comment:
Move this advice to lesson and remove the check for understanding header, please.

Reply:
Besides addressing our review comments, please be sure to see my comment in your slideshow, and follow the stying guidelines found in #600 . Also, we have several outlines on testing - search the repo for anything with “testing” in the file name, and consider adding them to ‘prerequisites’.

Reply:
Those things are cleared in the ppt file @alodahl and DevOps doesn't requires such prerequisites.